src/services/user.py

```
@@ -151,14 +151,19 @@ async def create_user(self, body: BodyCreateUser) -> UserResponse:
         logger.info(f"info_type=user_created ; user_id={user.id}")
         return UserResponse.model_validate(user)
 
-    def get_user_by_id(self, id: str) -> UserResponse:
+    def get_user_by_id(self, current_user: User, id: str) -> UserResponse:
```

Ruff (0.15.5)

[error] Function argument `id` is shadowing a Python builtin

(A002)